src/test/java/ai/elimu/dao/WordDaoTest.java

```diff
@@ -0,0 +1,51 @@
+package ai.elimu.dao;
+
+import ai.elimu.model.content.Word;
+import ai.elimu.model.v2.enums.content.WordType;
+import junit.framework.TestCase;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+
+@RunWith(SpringJUnit4ClassRunner.class)
+@ContextConfiguration("file:src/main/webapp/WEB-INF/spring/applicationContext-jpa.xml")
```

Comment:
Ref https://github.com/elimu-ai/webapp/pull/1601/files#diff-be21626776e7ac041ba6fad0d43302d787d5245997b8b325d8c983f837ac19a4

```suggestion
@ContextConfiguration(locations={
    "file:src/main/webapp/WEB-INF/spring/applicationContext.xml",
    "file:src/main/webapp/WEB-INF/spring/applicationContext-jpa.xml"
})
```